src/ceph.in

```diff
@@ -134,10 +134,19 @@ import subprocess
 from ceph_argparse import \
     concise_sig, descsort_key, parse_json_funcsigs, \
     matchnum, validate_command, find_cmd_target, \
-    send_command, json_command, run_in_thread
+    json_command, run_in_thread
 
 from ceph_daemon import admin_socket, DaemonWatcher, Termsize
 
+try:
+    errno.ETIMEOUT
```

Comment:
On my system there's an `errno.ETIMEDOUT` (with the D in the middle)

Reply:
no wonder i need to monkey patch errno here. but in my ipython, errno.E<TAB> always shows me something!